src/Data/Parameterized/ClassesC.hs

```diff
@@ -20,6 +20,9 @@ Note that there is still some ambiguity around naming conventions, see
 {-# LANGUAGE Trustworthy #-}
 {-# LANGUAGE TypeOperators #-}
 
+-- See https://github.com/GaloisInc/parameterized-utils/issues/149
+{-# OPTIONS_GHC -Wno-trustworthy-safe #-}
```

Comment:
Personally, `-Wtrustworthy-safe` is among the warnings that I would elect to disable globally, since I rarely find the amount of effort it causes to fix these warnings to be worth the hassle. (I'm fine with just disabling it per-module as is done here, however.)

Reply:
Not only is it not *worth* fixing, it's actually *problematic* to fix, as seen in #149...!

Reply:
I would vote for not accepting any more modules with `Trustworthy` nor `Safe` markers in the first place.

Reply:
Switched to `-Wdefault`, which does not include this warning. Left this line in, because `-Wall` is already enabled in the Cabal file.